docs/conf.py

```diff
@@ -46,6 +46,8 @@
 extlinks = {
     'commit': ('https://github.com/django/django/commit/%s', ''),
     'cve': ('https://nvd.nist.gov/view/vuln/detail?vulnId=%s', 'CVE-'),
+    # A file or directory. GitHub redirects from blob to tree if needed.
+    'source': ('https://github.com/django/django/blob/master/%s', ''),
```

Comment:
I wonder if we should use `blob/stable/N.M.x` or `blob/N.M.X` when docs are built for a specific version (e.g. `blob/stable/2.1.x` or `blob/2.1.1`). Is the version specifier even available during builds without having to rely on `git describe`?

Reply:
That was my plan. We can use `version + '.x'`.